### PR TITLE
Code review phase 1

### PR DESCRIPTION
**Pull request by vishu1994**
I have fixed the issues that were discussed.KIndly refer trello for more updates

### CODE REVIEW

File: domain.yml [hunk 1]
```diff
@@ -14,12 +14,15 @@ entities:
 slots:
   cuisine:
     type: text
-  location:
+  latitude:
+    type: text
```
Comment (kothiyayogesh) [inline domain.yml]: Check if slot type of float value works.

File: domain.yml [hunk 2]
```diff
@@ -14,12 +14,15 @@ entities:
...
+  location_name:
+    type: text
+  longitude:
     type: text
 templates:
   utter_greet:
-  - text: Hey!! I can help you to find restaurants based your preferred location and
-      cuisine.
```
Comment (kothiyayogesh) [inline domain.yml]: Keep response: Hey!! I can help you to find restaurants based on your preferred location and cuisine.

File: data/stories.md [hunk 3]
```diff
@@ -13,7 +13,9 @@
 * telling_location
     - utter_affirm_location
     - action_set_location
-    - slot{"location":"Hyderabad"}
+    - slot{"location_name":"Hyderabad"}
+    - slot{"latitude":17.3659992218}
```
[the remark this answers is not in the document]
Comment (vishu1994) [reply data/stories.md]: @kothiyayogesh  i will see this

File: domain.yml (hunk 1 above)
Comment (vishu1994) [reply domain.yml]: ok

**Pull request merged**